packages/eslint-config-terra/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@cerner/eslint-config-terra",
+  "version": "3.10.0",
+  "description": "Terra Eslint Config",
+  "main": "eslint.config.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-toolkit.git",
+    "directory": "packages/eslint-config-terra"
+  },
+  "keywords": [
+    "eslint",
+    "eslint-config",
+    "Cerner",
+    "Terra"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-toolkit/issues"
+  },
+  "engines": {
+    "node": "^8.10.0 || ^10.13.0 || >=11.10.1"
+  },
+  "homepage": "https://github.com/cerner/terra-toolkit",
+  "scripts": {
+    "lint": "eslint --ext .js,.jsx ."
+  },
+  "dependencies": {
```

Comment:
were we able to resolve this issue? https://github.com/cerner/terra-core/blob/master/package.json#L40-L48

Reply:
These aren't needed going forward.  I removed them in this PR: https://github.com/cerner/terra-core/pull/2948 until (if) we move doc template to use mdx syntax.